Makefile

```diff
@@ -67,8 +67,9 @@ secvarctl: $(OBJ)
 %.o: %.c
 	$(CC) $(CFLAGS) $(_CFLAGS) -c  $< -o $@
 
+clean:OBJ += crypto/crypto-openssl.o
 clean:
-	rm -f $(OBJ) secvarctl 
+	rm -f $(OBJ) secvarctl
```

Comment:
I wonder if it's worth following e.g. skiboot and just `rm` ing all .o and .d files anywhere they exist.

Reply:
Skiboot:
```
clean:
        $(RM) *.[odsa] $(SUBDIRS:%=%/*.[odsa])
```

Reply:
going to close this PR. Cleanup has been fixed in a commit in your recent PR regarding skiboot organization [here](https://github.com/open-power/secvarctl/pull/14) .